worlds/kh1/docs/kh1_en.md

```diff
@@ -3,35 +3,35 @@
 ##Setting up the required mods
 1. Install OpenKH and the LUA Backend.
 
-    Download the latest release of OpenKH here: https://github.com/OpenKH/OpenKh/releases/tag/latest
+    Download the latest release of OpenKH [here](https://github.com/OpenKH/OpenKh/releases/tag/latest)
```

Comment:
```suggestion
    Download the [latest release of OpenKH](https://github.com/OpenKH/OpenKh/releases/tag/latest)
```

Very minor and maybe you don't like it this way, so feel free to ignore

Reply:
I do like this better actually, thanks!